packages/docker/src/dockerCommands/container.ts

```diff
@@ -403,11 +408,16 @@ export async function containerRun(
   }
   if (args.environmentVariables) {
     for (const [key] of Object.entries(args.environmentVariables)) {
-      dockerArgs.push('-e')
-      dockerArgs.push(key)
+      dockerArgs.push('-e', key)
     }
   }
 
+  dockerArgs.push('-e', 'GITHUB_ACTIONS=true')
```

Comment:
What if `podContainer.env['GITHUB_ACTIONS']` already exists? Should we manage it as with `CI` to prevent duplication?